x-pack/plugins/enterprise_search/public/applications/app_search/components/synonyms/index.ts

```diff
@@ -6,3 +6,4 @@
  */
 
 export { SYNONYMS_TITLE } from './constants';
+export { Synonyms } from './synonyms';
```

Comment:
This word is tripping my brain up and I swear it never looks spelled correctly no matter how many times I type it. I guarantee I'll misspell it at least once while migrating so please feel free to holler if you see something 🦅